bundles/binding/org.openhab.binding.iec6205621meter/src/main/java/org/openhab/binding/iec6205621meter/internal/Iec6205621MeterBinding.java

```diff
@@ -177,11 +177,6 @@ protected void removeBindingProvider(Iec6205621MeterBindingProvider bindingProvi
     @Override
     public void updated(Dictionary<String, ?> config) throws ConfigurationException {
 
-        if (config == null || config.isEmpty()) {

```

Comment:
> Rather pointless to claim the config is required if it can be passed in as null isn't it?

Yes indeed.  My advice to leave the null check might be more superstition based on fuzzy past observations than how any OSGi SCR mechanisms actually work when configured properly.  Unless it's completely without merit, this kind of "defensive programming" can just avoid stupid frustration and time wasting later.